tests/test_container.py

```diff
@@ -73,3 +74,17 @@ def test_scene_remove():
     """Test Scene.remove()."""
     scene = Scene()
     container_remove(scene, lambda: scene.mobjects)
+
+
+def test_value_tracker_increment_value():
+    """Test ValueTracker.increment_value()"""
+    tracker = ValueTracker(0.0)
+    tracker.increment_value(10.0)
+    assert tracker.get_value() == 10.0
```

Comment:
**Issue : ** The problem here is that you are using `ValueTracker.get_value` method as an assertion, but this method does no have tests. Could you create a new test of this method ? (a quick one, like the two you wrote).